libsplinter/src/auth/oauth/mod.rs

```diff
@@ -118,7 +127,12 @@ impl OAuthClient {
     }
 
     /// Generates the URL that the end user should be redirected to for authorization
-    pub fn get_authorization_url(&self) -> Result<String, OAuthClientError> {
+    ///
+    /// # Arguments
+    ///
+    /// * `redirect_url` - The endpoint that Splinter will redirect to after it has completed
+    ///   authorization and the code exchange
+    pub fn get_authorization_url(&self, redirect_url: String) -> Result<String, OAuthClientError> {
```

Comment:
Can we rename this to `client_redirect_url` to clearly distinguish it from the OAuth redirect URL?

Reply:
Updated